lib/content-services/src/lib/aspect-list/aspect-list-dialog.component.html

```diff
@@ -1,7 +1,7 @@
-<header mat-dialog-title data-automation-id="aspect-list-dialog-title" class="adf-aspect-list-dialog">
-    <h4 class="adf-aspect-list-dialog-title">{{title | translate}}</h4>
+<div mat-dialog-title data-automation-id="aspect-list-dialog-title" class="adf-aspect-list-dialog">
+    <h1 class="adf-aspect-list-dialog-title">{{title | translate}}</h1>
```

Comment:
does it still preserve the font size as before?

Reply:
Yes it looks the same way as before.